src/activities/Elsa.Activities.SQL/Elsa.Activities.ExecuteSqlServerQuery.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <Import Project="..\..\..\common.props" />
+    <Import Project="..\..\..\configureawait.props" />
+    
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;netcoreapp3.1</TargetFrameworks>
+      <Description>
+          Elsa is a set of workflow libraries and tools that enable lean and mean workflowing capabilities in any .NET Core application.
+          This package provides activities to run SQL queries on given database using an connaction string.
+      </Description>
+      <PackageTags>elsa, workflows</PackageTags>
+  </PropertyGroup>
+    
+  <ItemGroup>
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.3" />
```

Comment:
Please use `Microsoft.Data.SqlClient` instead.

Refer this post: https://devblogs.microsoft.com/dotnet/introducing-the-new-microsoftdatasqlclient/